src/pages/tracked-contracts/TrackedContractsList.tsx

```diff
@@ -28,38 +29,15 @@ const TrackedContractsList: React.FC<{}> = () => {
     setChains(mockDataChain as Chain[]);
   }, []);
 
+  // Filter out by the search but also keep the current active in the search
   const allChains = chains
-    .filter((data) => {
-      if (!searchChain) {
-        return data;
-      }
-      if (data.name.includes(searchChain.trim().toLocaleLowerCase())) {
-        return data;
-      }
-      return null;
-    })
-    .map((chain) => {
-      return (
-        <div
-          className={`focus:outline-none py-4 capitalize cursor-pointer ${
-            chain.name === activeChain?.name ? 'font-bold text-primary' : ''
-          }`}
-          role="button"
-          onClick={() => setActiveChain(chain)}
-          onKeyPress={() => setActiveChain(chain)}
-          tabIndex={0}
-        >
-          {chain.name}
-        </div>
-      );
-    });
+    .filter((chain) => chain.name.includes(searchChain.trim().toLocaleLowerCase()) || activeChain?.name === chain.name)
```

Comment:
Just this filter works actually no need to check for if defined or null. Also added the fact that if a chan is already active you might still want to see it despite searching